src/styles/globals.css

```
@@ -0,0 +1,156 @@
+  --sidebar-primary-foreground: oklch(0.985 0.002 247.839);
+  --sidebar-accent: oklch(0.278 0.033 256.848);
+  --sidebar-accent-foreground: oklch(0.985 0.002 247.839);
+  --sidebar-border: oklch(1 0 0 / 10%);
+  --sidebar-ring: oklch(0.551 0.027 264.364);
+}
+
+@layer base {
+  * {
+    @apply border-border outline-ring/50;
+  }
+  :where(:not(:has([class*=" text-"]), ::has([class^="text-"]))) {
```

Biome (2.3.13)

[error] Unexpected unknown pseudo-elements: has([class^="text-"])

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)